packages/gatsby-theme-newrelic/src/components/TutorialSteps/TutorialStep.js

```diff
@@ -0,0 +1,101 @@
+import React, { Children } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import { isMdxType } from '../../utils/mdx';
+
+const ALLOWED_HEADINGS = ['h2', 'h3', 'h4'];
+
+const TutorialStep = ({ children, stepNumber, totalSteps }) => {
+  children = Children.toArray(children);
+
+  const title = ALLOWED_HEADINGS.some((level) => isMdxType(children[0], level))
+    ? children[0]
+    : null;
```

Comment:
for some reason i'm not super clear on what we're doing here or if it's necessary? if anything, might be worth documenting for writers if they want to set a section title.

if the immediate child of the `Step` component is a `<h2>`, `<h3>`, `<h4>` set it as the title of the section. if not, do nothing and move on.

weirdly, if you use markdown syntax for immediate child like so:

```
<Step>
## Heading 2

More content
```

results in this:
![2023-01-19_11-58-12](https://user-images.githubusercontent.com/2952843/213547060-87789aab-e157-4125-af31-a2d7e526a41a.png)


but doing this:

```
<Step>
<h2>Heading 2</h2>

More content
```

results in this, which is more like what's expected?

![2023-01-19_11-57-49](https://user-images.githubusercontent.com/2952843/213547107-e63d7322-3081-4b44-aa89-fe4a60b30645.png)